### PR TITLE
Add flow type for cssnano

### PR DESCRIPTION
**Pull request by thebriando**
# ↪️ Pull Request
Adds flow type libary to `cssnano` package.

## 🚨 Test instructions
Run `yarn flow check` and verify that there are no flow errors.

### CODE REVIEW

File: flow-libs/cssnano.js.flow [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// Derived from the source of cssnano located at
+// https://github.com/cssnano/cssnano and its types at
+// https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cssnano/index.d.ts
+
+declare module 'cssnano' {
+  declare interface CSSNanoOptions {
+    configFile?: string;
+    preset?: [string, object] | string;
+  }
+  declare module.exports: ?CSSNano;
```
Comment (mischnic) [inline flow-libs/cssnano.js.flow:12]: Is  `CSSNano` defined anywhere? It probably silently became `any`

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 4.10s | -135.00ms |
| Cached | 795.00ms | +7.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 62.00ms | +16.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 56.00ms | -7.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 83.00ms | +25.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 57.00ms | -26.00ms 🚀 |
| dist/legacy/index.7e103b24.js | 1.07kb | +0.00b | 91.00ms | +7.00ms ⚠️ |
| dist/legacy/index.507458dd.css | 77.00b | +0.00b | 89.00ms | +5.00ms ⚠️ |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 23.01s | -398.00ms |
| Cached | 21.86s | +21.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/NotFound.5d9b792b.js | 532.00b | +0.00b | 299.00ms | +23.00ms ⚠️ |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 84.00ms | +6.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.62e59ad1.js | 4.22kb | +0.00b | 87.00ms | -24.00ms 🚀 |
| dist/UserProfile.7223b396.js | 1.70kb | +0.00b | 87.00ms | -24.00ms 🚀 |
| dist/NotFound.5d9b792b.js | 532.00b | +0.00b | 48.00ms | -265.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 32.00ms | -4.00ms 🚀 |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.13m | +217.00ms |
| Cached | 3.00s | +23.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/popup.6dabe30d.js | 169.35kb | +0.00b | 23.45s | -2.11s 🚀 |
| dist/EmojiPickerComponent.4faa5472.js | 139.08kb | +0.00b | 9.71s | -1.24s 🚀 |
| dist/media-viewer.a564b877.js | 72.71kb | +0.00b | 11.73s | -1.35s 🚀 |
| dist/Modal.804f32ae.js | 39.02kb | +0.00b | 4.37s | -898.00ms 🚀 |
| dist/component.effe7aa0.js | 30.87kb | +0.00b | 2.00s | -474.00ms 🚀 |
| dist/esm.d4fe111d.js | 27.71kb | +0.00b | 8.44s | -2.50s 🚀 |
| dist/component.7d45f63d.js | 22.40kb | +0.00b | 4.37s | -898.00ms 🚀 |
| dist/DatePicker.27ec5589.js | 21.16kb | +0.00b | 9.70s | -1.23s 🚀 |
| dist/js.aaf5ecd1.js | 16.55kb | +0.00b | 4.37s | -897.00ms 🚀 |
| dist/dropzone.27b639af.js | 16.03kb | +0.00b | 23.72s | -22.01s 🚀 |
| dist/workerHasher.8a9cd633.js | 11.90kb | +0.00b | 9.70s | +665.00ms ⚠️ |
| dist/card.f50bfb7e.js | 5.71kb | +0.00b | 11.31s | -1.28s 🚀 |
| dist/media-viewer.b1642a16.js | 4.01kb | +0.00b | 13.95s | -1.35s 🚀 |
| dist/png-chunks-extract.1a612bb6.js | 3.55kb | +0.00b | 4.37s | -898.00ms 🚀 |
| dist/dropzone.ab62bdb4.js | 3.44kb | +0.00b | 23.72s | -22.01s 🚀 |
| dist/Modal.99a80838.js | 3.15kb | +0.00b | 2.37s | -842.00ms 🚀 |
| dist/16.0cf912f4.js | 2.49kb | +0.00b | 2.00s | -468.00ms 🚀 |
| dist/card.ac215339.js | 2.15kb | +0.00b | 13.96s | -1.35s 🚀 |
| dist/date.90ec1302.js | 1.96kb | +0.00b | 5.22s | -678.00ms 🚀 |
| dist/images.c69c067f.js | 1.90kb | +0.00b | 5.22s | -828.00ms 🚀 |
| dist/feedback.a4a4dbc3.js | 1.86kb | +0.00b | 9.70s | -1.24s 🚀 |
| dist/16.2daaa966.js | 1.86kb | +0.00b | 4.72s | -1.05s 🚀 |
| dist/browser.d4b2ac0a.js | 1.82kb | +0.00b | 46.15s | -30.62s 🚀 |
| dist/workerHasher.6aa37edc.js | 1.75kb | +0.00b | 9.66s | +685.00ms ⚠️ |
| dist/workerHasher.f631e5e0.js | 1.75kb | +0.00b | 1.26m | +28.66s ⚠️ |
| dist/list-number.d6f09d2f.js | 1.68kb | +0.00b | 5.42s | -628.00ms 🚀 |
| dist/status.09dffefc.js | 1.68kb | +0.00b | 5.77s | -529.00ms 🚀 |
| dist/code.b1fce661.js | 1.61kb | +0.00b | 4.95s | -949.00ms 🚀 |
| dist/link.18257706.js | 1.53kb | +0.00b | 5.42s | -628.00ms 🚀 |
| dist/heading6.ee7c597d.js | 1.53kb | +0.00b | 9.70s | +3.29s ⚠️ |
| dist/heading3.5a833b7b.js | 1.51kb | +0.00b | 8.04s | +1.63s ⚠️ |
| dist/16.542d3ab7.js | 1.51kb | +0.00b | 4.52s | -1.11s 🚀 |
| dist/16.72d89b1f.js | 1.46kb | +0.00b | 4.51s | -1.09s 🚀 |
| dist/emoji.a556f6a6.js | 1.45kb | +0.00b | 5.22s | -828.00ms 🚀 |
| dist/16.35c1a785.js | 1.45kb | +0.00b | 2.00s | -473.00ms 🚀 |
| dist/16.e7cb0586.js | 1.44kb | +0.00b | 4.94s | -947.00ms 🚀 |
| dist/16.388091fb.js | 1.40kb | +0.00b | 4.94s | -2.04s 🚀 |
| dist/heading5.368d12b0.js | 1.40kb | +0.00b | 8.04s | +1.63s ⚠️ |
| dist/expand.abd57bf3.js | 1.38kb | +0.00b | 9.70s | -1.24s 🚀 |
| dist/16.493f9d2a.js | 1.36kb | +0.00b | 4.72s | -1.05s 🚀 |
| dist/16.1e8baed6.js | 1.34kb | +0.00b | 2.00s | -469.00ms 🚀 |
| dist/heading2.de835502.js | 1.33kb | +0.00b | 5.77s | -642.00ms 🚀 |
| dist/16.0c3e9d9e.js | 1.32kb | +0.00b | 4.72s | -1.05s 🚀 |
| dist/16.56cee737.js | 1.31kb | +0.00b | 4.94s | -947.00ms 🚀 |
| dist/media-card-analytics-error-boundary.2fc3699c.js | 1.30kb | +0.00b | 11.31s | -1.28s 🚀 |
| dist/mention.0381ad00.js | 1.29kb | +0.00b | 5.42s | -743.00ms 🚀 |
| dist/heading4.abd6ed5d.js | 1.29kb | +0.00b | 8.04s | +1.63s ⚠️ |
| dist/Modal.c5a75f57.js | 1.28kb | +0.00b | 4.37s | -897.00ms 🚀 |
| dist/16.a785c6a9.js | 1.27kb | +0.00b | 2.00s | -473.00ms 🚀 |
| dist/layout.6fc445d4.js | 1.27kb | +0.00b | 5.42s | -623.00ms 🚀 |
| dist/16.350f9e86.js | 1.27kb | +0.00b | 4.72s | -1.05s 🚀 |
| dist/16.d2bd05f5.js | 1.26kb | +0.00b | 4.51s | -1.09s 🚀 |
| dist/16.0c59612d.js | 1.26kb | +0.00b | 4.51s | -1.09s 🚀 |
| dist/16.0a326f87.js | 1.26kb | +0.00b | 4.52s | -1.11s 🚀 |
| dist/divider.2e530f7c.js | 1.25kb | +0.00b | 5.22s | -1.79s 🚀 |
| dist/quote.df9cfef0.js | 1.25kb | +0.00b | 5.77s | -529.00ms 🚀 |
| dist/component.9c6ba2f0.js | 1.23kb | +0.00b | 2.37s | -834.00ms 🚀 |
| dist/action.b6fa0d15.js | 1.23kb | +0.00b | 4.95s | -946.00ms 🚀 |
| dist/16.76d22cc2.js | 1.23kb | +0.00b | 2.37s | -842.00ms 🚀 |
| dist/decision.0dfb05c0.js | 1.21kb | +0.00b | 5.22s | -1.79s 🚀 |
| dist/panel-warning.d1deb7d8.js | 1.21kb | +0.00b | 5.55s | -627.00ms 🚀 |
| dist/16.b4718b96.js | 1.18kb | +0.00b | 4.72s | -1.05s 🚀 |
| dist/list.fc45edf6.js | 1.18kb | +0.00b | 5.42s | -1.77s 🚀 |
| dist/heading1.cc39194f.js | 1.18kb | +0.00b | 5.77s | -530.00ms 🚀 |
| dist/panel-error.d22bf8a5.js | 1.11kb | +0.00b | 5.55s | -611.00ms 🚀 |
| dist/panel.b435337e.js | 1.10kb | +0.00b | 5.55s | -743.00ms 🚀 |
| dist/table.964f293b.js | 1.09kb | +0.00b | 5.77s | -530.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.133a9728.js | 1.05kb | +0.00b | 23.72s | -22.01s 🚀 |
| dist/panel-success.0b507c40.js | 1.05kb | +0.00b | 5.55s | -628.00ms 🚀 |
| dist/media-card-analytics-error-boundary.64bc429c.js | 1.05kb | +0.00b | 11.31s | -1.28s 🚀 |
| dist/panel-note.07086709.js | 1.05kb | +0.00b | 5.55s | -611.00ms 🚀 |
| dist/media-viewer-analytics-error-boundary.c9f83b12.js | 995.00b | +0.00b | 13.96s | -1.35s 🚀 |
| dist/simpleHasher.4e275b4f.js | 755.00b | +0.00b | 9.66s | +644.00ms ⚠️ |
| dist/index.html | 119.00b | +0.00b | 3.34s | -494.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/pdfRenderer.50b8173f.js | 1.11mb | +0.00b | 639.00ms | +41.00ms ⚠️ |
| dist/editorView.267a5df1.js | 587.39kb | +0.00b | 673.00ms | +42.00ms ⚠️ |
| dist/popup.3b70ca2b.js | 169.21kb | +0.00b | 681.00ms | +70.00ms ⚠️ |
| dist/EmojiPickerComponent.4faa5472.js | 139.08kb | +0.00b | 638.00ms | +67.00ms ⚠️ |
| dist/Toolbar.cd51d763.js | 99.13kb | +0.00b | 675.00ms | +49.00ms ⚠️ |
| dist/media-viewer.0aa3923d.js | 72.60kb | +0.00b | 639.00ms | +42.00ms ⚠️ |
| dist/card.33239941.js | 53.89kb | +0.00b | 642.00ms | +45.00ms ⚠️ |
| dist/card.6faf41b0.js | 51.65kb | +0.00b | 666.00ms | +52.00ms ⚠️ |
| dist/Modal.f72d1800.js | 38.79kb | +0.00b | 554.00ms | +106.00ms ⚠️ |
| dist/component.effe7aa0.js | 30.87kb | +0.00b | 533.00ms | +101.00ms ⚠️ |
| dist/esm.d4fe111d.js | 27.71kb | +0.00b | 639.00ms | +65.00ms ⚠️ |
| dist/component.7d45f63d.js | 22.40kb | +0.00b | 543.00ms | +95.00ms ⚠️ |
| dist/DatePicker.7146e115.js | 21.03kb | +0.00b | 638.00ms | +67.00ms ⚠️ |
| dist/smartMediaEditor.dafbb08e.js | 16.69kb | +0.00b | 673.00ms | +42.00ms ⚠️ |
| dist/js.aaf5ecd1.js | 16.55kb | +0.00b | 543.00ms | +95.00ms ⚠️ |
| dist/dropzone.8003c168.js | 15.96kb | +0.00b | 680.00ms | +63.00ms ⚠️ |
| dist/ui.9353c8f7.js | 14.11kb | +0.00b | 641.00ms | +44.00ms ⚠️ |
| dist/workerHasher.8a9cd633.js | 11.90kb | +0.00b | 627.00ms | +53.00ms ⚠️ |
| dist/component.adc3ddbc.js | 6.27kb | +0.00b | 543.00ms | +78.00ms ⚠️ |
| dist/card.f50bfb7e.js | 5.71kb | +0.00b | 639.00ms | +42.00ms ⚠️ |
| dist/media-viewer.b1642a16.js | 4.01kb | +0.00b | 664.00ms | +59.00ms ⚠️ |
| dist/png-chunks-extract.1a612bb6.js | 3.55kb | +0.00b | 543.00ms | +95.00ms ⚠️ |
| dist/index.c1492881.css | 3.46kb | +0.00b | 680.00ms | +49.00ms ⚠️ |
| dist/dropzone.ab62bdb4.js | 3.44kb | +0.00b | 680.00ms | +63.00ms ⚠️ |
| dist/clipboard.d10471c4.js | 2.97kb | +0.00b | 680.00ms | +63.00ms ⚠️ |
| dist/16.0cf912f4.js | 2.49kb | +0.00b | 533.00ms | +101.00ms ⚠️ |
| dist/ResourcedEmojiComponent.e6000ed9.js | 2.15kb | +0.00b | 629.00ms | +32.00ms ⚠️ |
| dist/card.ac215339.js | 2.15kb | +0.00b | 664.00ms | +53.00ms ⚠️ |
| dist/date.90ec1302.js | 1.96kb | +0.00b | 584.00ms | +65.00ms ⚠️ |
| dist/images.c69c067f.js | 1.90kb | +0.00b | 587.00ms | +61.00ms ⚠️ |
| dist/feedback.a4a4dbc3.js | 1.86kb | +0.00b | 621.00ms | +50.00ms ⚠️ |
| dist/16.2daaa966.js | 1.86kb | +0.00b | 568.00ms | +87.00ms ⚠️ |
| dist/browser.d4b2ac0a.js | 1.82kb | +0.00b | 666.00ms | +49.00ms ⚠️ |
| dist/16.72628d37.js | 1.79kb | +0.00b | 560.00ms | +94.00ms ⚠️ |
| dist/workerHasher.6aa37edc.js | 1.75kb | +0.00b | 626.00ms | +52.00ms ⚠️ |
| dist/workerHasher.f631e5e0.js | 1.75kb | +0.00b | 670.00ms | +47.00ms ⚠️ |
| dist/list-number.d6f09d2f.js | 1.68kb | +0.00b | 594.00ms | +56.00ms ⚠️ |
| dist/status.09dffefc.js | 1.68kb | +0.00b | 618.00ms | +62.00ms ⚠️ |
| dist/code.b1fce661.js | 1.61kb | +0.00b | 584.00ms | +65.00ms ⚠️ |
| dist/link.18257706.js | 1.53kb | +0.00b | 594.00ms | +67.00ms ⚠️ |
| dist/heading6.ee7c597d.js | 1.53kb | +0.00b | 621.00ms | +50.00ms ⚠️ |
| dist/heading3.5a833b7b.js | 1.51kb | +0.00b | 620.00ms | +55.00ms ⚠️ |
| dist/16.542d3ab7.js | 1.51kb | +0.00b | 562.00ms | +82.00ms ⚠️ |
| dist/16.f683480d.js | 1.46kb | +0.00b | 561.00ms | +95.00ms ⚠️ |
| dist/emoji.a556f6a6.js | 1.45kb | +0.00b | 587.00ms | +61.00ms ⚠️ |
| dist/16.35c1a785.js | 1.45kb | +0.00b | 533.00ms | +101.00ms ⚠️ |
| dist/16.e7cb0586.js | 1.44kb | +0.00b | 584.00ms | +81.00ms ⚠️ |
| dist/16.388091fb.js | 1.40kb | +0.00b | 580.00ms | +78.00ms ⚠️ |
| dist/heading5.368d12b0.js | 1.40kb | +0.00b | 621.00ms | +50.00ms ⚠️ |
| dist/expand.abd57bf3.js | 1.38kb | +0.00b | 622.00ms | +51.00ms ⚠️ |
| dist/16.493f9d2a.js | 1.36kb | +0.00b | 578.00ms | +76.00ms ⚠️ |
| dist/16.1e8baed6.js | 1.34kb | +0.00b | 533.00ms | +101.00ms ⚠️ |
| dist/heading2.de835502.js | 1.33kb | +0.00b | 624.00ms | +59.00ms ⚠️ |
| dist/16.0c3e9d9e.js | 1.32kb | +0.00b | 579.00ms | +77.00ms ⚠️ |
| dist/16.56cee737.js | 1.31kb | +0.00b | 584.00ms | +66.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.2fc3699c.js | 1.30kb | +0.00b | 639.00ms | +51.00ms ⚠️ |
| dist/mention.0381ad00.js | 1.29kb | +0.00b | 597.00ms | +53.00ms ⚠️ |
| dist/heading4.abd6ed5d.js | 1.29kb | +0.00b | 626.00ms | +55.00ms ⚠️ |
| dist/Modal.c5a75f57.js | 1.28kb | +0.00b | 543.00ms | +95.00ms ⚠️ |
| dist/16.a785c6a9.js | 1.27kb | +0.00b | 533.00ms | +101.00ms ⚠️ |
| dist/layout.6fc445d4.js | 1.27kb | +0.00b | 588.00ms | +62.00ms ⚠️ |
| dist/16.350f9e86.js | 1.27kb | +0.00b | 564.00ms | +83.00ms ⚠️ |
| dist/16.d2bd05f5.js | 1.26kb | +0.00b | 560.00ms | +94.00ms ⚠️ |
| dist/16.0c59612d.js | 1.26kb | +0.00b | 561.00ms | +81.00ms ⚠️ |
| dist/16.0a326f87.js | 1.26kb | +0.00b | 564.00ms | +83.00ms ⚠️ |
| dist/16.540cfafc.js | 1.26kb | +0.00b | 561.00ms | +96.00ms ⚠️ |
| dist/divider.2e530f7c.js | 1.25kb | +0.00b | 587.00ms | +61.00ms ⚠️ |
| dist/quote.df9cfef0.js | 1.25kb | +0.00b | 611.00ms | +66.00ms ⚠️ |
| dist/action.b6fa0d15.js | 1.23kb | +0.00b | 584.00ms | +65.00ms ⚠️ |
| dist/16.76d22cc2.js | 1.23kb | +0.00b | 557.00ms | +30.00ms ⚠️ |
| dist/decision.0dfb05c0.js | 1.21kb | +0.00b | 586.00ms | +67.00ms ⚠️ |
| dist/panel-warning.d1deb7d8.js | 1.21kb | +0.00b | 603.00ms | +58.00ms ⚠️ |
| dist/16.b4718b96.js | 1.18kb | +0.00b | 579.00ms | +77.00ms ⚠️ |
| dist/list.fc45edf6.js | 1.18kb | +0.00b | 594.00ms | +55.00ms ⚠️ |
| dist/heading1.cc39194f.js | 1.18kb | +0.00b | 618.00ms | +53.00ms ⚠️ |
| dist/panel-error.d22bf8a5.js | 1.11kb | +0.00b | 599.00ms | +54.00ms ⚠️ |
| dist/panel.b435337e.js | 1.10kb | +0.00b | 608.00ms | +63.00ms ⚠️ |
| dist/table.964f293b.js | 1.09kb | +0.00b | 618.00ms | +62.00ms ⚠️ |
| dist/media-picker-analytics-error-boundary.133a9728.js | 1.05kb | +0.00b | 680.00ms | +63.00ms ⚠️ |
| dist/panel-success.0b507c40.js | 1.05kb | +0.00b | 603.00ms | +58.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.64bc429c.js | 1.05kb | +0.00b | 639.00ms | +51.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.e9c7a84a.js | 1.05kb | +0.00b | 670.00ms | +47.00ms ⚠️ |
| dist/panel-note.07086709.js | 1.05kb | +0.00b | 602.00ms | +57.00ms ⚠️ |
| dist/media-viewer-analytics-error-boundary.c9f83b12.js | 995.00b | +0.00b | 664.00ms | +58.00ms ⚠️ |
| dist/simpleHasher.a8bd8fa2.js | 755.00b | +0.00b | 670.00ms | +39.00ms ⚠️ |

</p></details>

<details><summary>Three.js ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 15.31s | -169.00ms |
| Cached | 688.00ms | -23.00ms |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/Three.js | 580.68kb | +0.00b | 95.00ms | -12.00ms 🚀 |

</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/285658517903245836)**

File: flow-libs/cssnano.js.flow (hunk 1 above)
Comment (thebriando) [reply flow-libs/cssnano.js.flow:12]: I only see it being used in `CSSNanoOptimizer.js`

Comment (mischnic) [reply flow-libs/cssnano.js.flow:12]: I mean you're using the identifier `CSSNano` in this file but it's not declared anywhere 😄

Comment (thebriando) [reply flow-libs/cssnano.js.flow:12]: Oh I see, yea I think that should be `CSSNanoOptions` 😄

Comment (mischnic) [reply flow-libs/cssnano.js.flow]: It does export some kind of function: https://cssnano.co/docs/getting-started/#using-postcss-cli

Comment (thebriando) [reply flow-libs/cssnano.js.flow]: Hmm are you referring to `preset`? I added it in `CSSNanoOptions`

Comment (mischnic) [reply flow-libs/cssnano.js.flow]: You're currently exporting the object itself. But it's a function with the options as a parameter:
```
declare module.exports: (?CSSNanoOptions) => mixed;
```

**Review by mischnic — APPROVED** [review]

**Pull request merged**